recipes/hint/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "2.0.1" %}
+
+package:
+  name: hint
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/d4/ba/0391dddacaa66b599cff112bc2bed7e0f915ef4aeaf6bf7619368eff3209/HiNT-Package-2.0.2.tar.gz
+  sha256: a57d9ed2e1954cd174aa9a0748f7af56d80c9cd9b4b4deb44a89cefbe4301ecd
+
+build:
+  number: 0
+  noarch: python
+  #script: $PYTHON -m pip install --no-deps --ignore-installed --no-cache-dir .
```

Comment:
@suwangbio is that not working? I would prefer this line and remove the buid.sh script altogether.